lib/modules/dosomething/dosomething_user/dosomething_user.address.inc

```diff
@@ -145,9 +149,35 @@ function dosomething_user_save_address_values($values) {
       ),
     );
   }
+
   user_save($account, $edit);
 }
 
+function dosomething_user_save_address_values_to_northstar($user, $values) {
+  $northstar_id = dosomething_user_get_northstar_id($user->uid);
+  $profile = [];
+  $submission = $values['user_address'];
+
+  // Map Signup Data Form fields to Northstar profile fields.
+  $address_fields = [
+    'thoroughfare' => 'addr_street1',
+    'premise' => 'addr_street2',
+    'locality' => 'addr_city',
+    'administrative_area' => 'addr_state',
+    'postal_code' => 'addr_zip',
+  ];
+
+  foreach ($address_fields as $drupal_key => $northstar_key) {
+    $field = $submission[$drupal_key];
+
+    if ($field) {
+      $profile[$northstar_key] = $field;
+    }
+  }
+
+  dosomething_northstar_client()->asClient()->put('v1/users/id/' . $northstar_id, $profile);
```

Comment:
Oh, we probably don't need to be requiring the `user` scope there. Let's fix that in Northstar.